themes/default/content/templates/serverless-application/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Serverless Templates
+title: Serverless Applications and Templates
```

Comment:
I think "Serverless Application Templates" (omit the "and") is more accurate here